client/src/Components/Alerts/AlertList.js

```diff
@@ -86,14 +88,14 @@ const AlertList = () => {
             </div>
           </div>
         </div>
-        {list.map(alert => (
+        {list.filter(alert => alert.creator == creatorId).map(alert => (
```

Comment:
this approach works well; however, I find it more readable to move more complex logic out of the html template and into the controller.

I suggest making this line:
`{alertsByCreator.map(alert => (`

and define `alertsByCreator` above where you're fetching the creatorId